### ISSUE TITLE
Alternative feature enhancement: Show what the entry should be the alternative to

### ISSUE THREAD

**Issue by Ferocit** [sabnzbd/sabnzbd#2793]
### Description

I really like the new "Alternative" label, unfortunately it's not always clear which other download it refers to. It would be great if we could somehow link the different entries. 

You could add an "original" label and add a number to each label. Or make it clickable and jump to the original. A tooltip with the title of the original entry could help, too.

**Comment by thezoggy** [comment]
doesnt it show the source is the url of where it came from (the indexer header url) ?

**Comment by Safihre** [comment]
Good feature request!

**Comment by Ferocit** [comment]
> doesnt it show the source is the url of where it came from (the indexer header url) when it lands in the history ? or are you asking about showing where it came while its still in the queue?

Yes, I'd like to keep my queue clean and therefor check for duplicates in the queue. The "alternative" label really helps, before I had to sort by name and by size to look for duplicates. 

If I keep them in the queue, then I just get a "ignoring duplicate nzb" for the file when it's their turn. Unfortunately it even looks like I got a few false positives for very long titles with minor differences.

**Comment by timbuckiii** [comment]
+1 to this feature request
